benchmarks/lasso/solvers/baseline.py

```diff
@@ -5,33 +5,34 @@
 
 
 class Solver(BaseSolver):
-    name = 'Baseline'
+    name = 'Baseline'  # proximal gradient, eventually accelerated
 
+    # any parameter defined here is accessible as a class attribute
     parameters = {'use_acceleration': [False, True]}
 
     def set_objective(self, X, y, lmbd):
         self.X, self.y, self.lmbd = X, y, lmbd
 
-        self.L = np.linalg.norm(self.X.dot(self.X.T), ord=2)
-
     def run(self, n_iter):
+        L = np.linalg.norm(self.X, ord=2) ** 2
+
         n_features = self.X.shape[1]
-        z_hat = np.zeros(n_features)
+        w = np.zeros(n_features)
```

Comment:
If we decide to go that way I would say that this need to be  changed globally in the benchmark. I think I used `z_hat` everywhere but we can do this in a separate PR.

Reply:
+1 to merge this one and do another PR